meson.build

```diff
@@ -102,6 +102,10 @@ if enable_recorder
   recorder_deps += [gst_dep, gst_base_dep, gtk_dep, x11_dep]
 endif
 
+cc = meson.get_compiler('c')
+
+m_dep = cc.find_library('m', required: false)
```

Comment:
Why is this required? I'd avoid diverging from the upstream build file if possible.

Reply:
Its required since EndlesShellFX depends on `libm`. I can probably get rid of EndlessShellFX now that we dont need to compile it separately but would rather do that in another story since it's a lot of churn.